target/linux/octeon/image/Makefile

```diff
@@ -47,4 +47,40 @@ define Device/ubnt_edgerouter-lite
 endef
 TARGET_DEVICES += ubnt_edgerouter-lite
 
+#
+# Itus Network Shield Profiles
+#
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itusrouter
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
+  BOARD_NAME := itusrouter
```

Comment:
I'm not too familiar with non-DTS targets. Do we still need BOARD_NAME here?

Reply:
it's carried thru the build system as $(board_name) AFAIK, at least, which it isn't a $SUBTARGET, in which case I could only get it to put "itus" (as the subtarget) instead of the BOARD_NAME defined.

Reply:
subtarget is a completely different thing.

BOARD_NAME is mostly an old mechanism for SUPPORTED_DEVICES and also influences some file names in the image assembly process.